src/main/java/com/gluonhq/richtext/undo/CommandManager.java

```diff
@@ -32,8 +34,11 @@ public void undo() {
     public void redo() {
         if (!redoStack.isEmpty()) {
             var cmd = redoStack.pop();
+            List<AbstractCommand<T>> redoStackCopy = new ArrayList<>(redoStack);
+            redoStack.clear();
             cmd.redo(context);
             undoStack.push(cmd);
+            redoStack.addAll(redoStackCopy);
```

Comment:
Why do we need that?  If command is redone, it is removed from the top stack.
What else is needed?

Reply:
See description of the issue. Redo stack gets cleared right after one redo call, which is wrong.

Reply:
The first time redo is called,`InsertTextCmd::doRedo` calls `insert` which calls `command::execute` that does 
`redoStack.clear()`.
Since `execute` can't be modified (to avoid clearing the redo stack on a redo operation), the proposed fix just stores the commands left on the redo stack and after performing the redo call, restores the redo stack with them.
Does it make sense to you?